pandas/io/parsers.py

```diff
@@ -879,7 +880,8 @@ def __init__(self, f, engine=None, **kwds):
         self._make_engine(self.engine)
 
     def close(self):
-        self._engine.close()
+        if self.engine != "arrow":
+            self._engine.close()
```

Comment:
Remind me why do we need this?  Add a comment to clarify.

Reply:
fixed